model/minsky.cc

```diff
@@ -255,27 +255,47 @@ namespace minsky
 //      throw runtime_error("cannot save to "+fileName);
   }
 
-  void Minsky::paste(float x, float y)
```

Comment:
Aww - I quite liked the idea of passing in coordinates for where to paste the contents...

Reply:
I also thought it was a good idea, and think I came close to getting it right but feel my understanding of how the coordinates are handled between TCL and Minsky is still on the low side.